### PR TITLE
@ #23 | k8s should run correctly

### PR DESCRIPTION
**Pull request by hieptranquoc**
connect #23 
please check this PR! Thank you

### CODE REVIEW

File: config_default.yaml [hunk 1]
```diff
@@ -87,4 +87,8 @@ teracy-dev-k8s:
     host_vars:
       ingress_nginx_enabled: "True"
       helm_enabled: "True"
-      helm_version: v2.11.0
+      helm_version: v2.12.1
+      disable_ipv6_dns: "True"
```
Comment (hoatle) [inline config_default.yaml:91]: it's better to move these stuff to teracy-dev-k8s extension instead.

**Review by hoatle — COMMENTED** [review]
@hieptranquoc please check my comments.

File: config_default.yaml (hunk 1 above)
Comment (hieptranquoc) [reply config_default.yaml:91]: @hoatle I applied your comment. Please check again at https://github.com/teracyhq-incubator/teracy-dev-k8s/pull/72! Thank you!

**Comment by hieptranquoc** [comment]
resolved : https://github.com/teracyhq-incubator/teracy-dev-k8s/pull/72

**Comment by hoatle** [comment]
fixed by https://github.com/teracyhq-incubator/teracy-dev-k8s/pull/72

**Pull request closed without merging**